tests/x11regressions/evolution/evolution_mail_ews.pm

```diff
@@ -0,0 +1,128 @@
+#Test Case #1503965: Evolution: Setup MS Exchange account
+use strict;
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $mailbox     = 'zzzSUSEExTest19@microfocus.com';
+    my $mail_passwd = 'P@$$w0rd2015';
+
+    mouse_hide(1);
+
+    # Clean and Start Evolution
+    x11_start_program("xterm -e \"killall -9 evolution; find ~ -name evolution | xargs rm -rf;\"");
+    x11_start_program("evolution");
+    if (check_screen "evolution-default-client-ask") {
+        assert_and_click "evolution-default-client-agree";
+    }
+
+    # Follow the wizard to setup mail account
+    assert_screen "test-evolution-1";
```

Comment:
Seems I didn't understand your suggestion well when I wrote my previous note. You suggested to match multiple possible needles in one assert_screen and handle the rare case in if condition, which would make the test faster in most cases.
Thanks a lot for the approach.